### ISSUE TITLE
Can't add platforms with Swarm node

### ISSUE THREAD

**Issue by ggarnier** [tsuru/tsuru#1612]
Currently if I have a Swarm node, but no Docker node, `tsuru platform-add <platform>` gives me this error:

```
Error: There is no docker node. Please list one in tsuru.conf or add one with `tsuru node-add`.
```

Swarm node should be capable of building platforms too.

**Comment by cezarsa** [comment]
I think this has been fixed, right @guilhermebr ?

**Comment by ggarnier** [comment]
@cezarsa this was fixed in tsuru/tsuru#1618. I'm just making a small adjustment in the integration test to close this issue.
